business/services.go

```diff
@@ -885,7 +888,7 @@ func (in *SvcService) GetServiceTracingName(ctx context.Context, cluster, namesp
 		return tracingName, nil
 	}
 
-	appLabelName := in.config.IstioLabels.AppLabelName
+	appLabelName, _ := in.config.GetAppLabelName(svc.Selectors) // Is it OK that this assumes the label is found?
```

Comment:
at minimum, you should log a message, even if at DEBUG level. We are going to want to see if this is not found when it happens.  Otherwise, this sounds like it could be an insidious problem to find.

Reply:
OK, I'll add a debug msg.